package.json

```diff
@@ -554,6 +553,7 @@
     "acorn": "^7.0.0",
     "assetgraph-builder": "^6.10.1",
     "autoprefixer": "^9.6.0",
+    "babel-eslint": "^10.0.3",
```

Comment:
Why do we need this dependency?

Reply:
I think this can be replaced with something in the `ecmaFeatures` config.  still, that should not be allowed except in our sources that use `import`

Reply:
otoh, it may be that eslint doesn't support `import()`?

Reply:
@boneskull - yup, eslint doesn't support `import()`. This is what happens when I remove `babel-eslint` from `.eslintrc.yml`:
```
> nps test
...
nps is executing `lint.code` : eslint . "bin/*"

/Users/giltayar/code/mocha/lib/esm-utils.js
  14:14  error  Parsing error: Unexpected token import
```

Reply:
Was this resolved with #4115?

Reply:
Unfortunately, no. `import()` is ES2018, and #4115 updated to ES2017, due to the need to support Node v8.